Import deployable tutorial (#495)
OctopusDeploy/blog: blog/2021-09/importing-a-deployable-tutorial/high-level-overview/high-level-overview.md

- Specific solutions to common customer problems, such as [Configure team members’ machines with Chocolatey, PowerShell, and Octopus Runbooks](https://octopus.com/blog/chocolatey-powershell-and-runbooks)
- An explanation of new features we're releasing, including [Announcing GitHub Actions for Octopus Deploy](https://octopus.com/blog/github-actions-for-octopus-deploy)
- Tool specific guides, such as [Deploying a Ruby web application](https://octopus.com/blog/deploying-ruby)
- Kubernetes deployments, for example this ebook, [10 pillars of pragmatic Kubernetes deployments](https://octopus.com/blog/10-pillars-kubernetes-deployments)

You'll also find release notes, company announcements, and more. Customers tell us they enjoy engaging with the blog and find it helpful and informative, and we hope you do too. 

[Visit the blog page](https://octopus.com/blog/) to see the latest posts and search the topics that interest you.

#### Slack

Our Community Slack allows users to interact with other Octopus Deploy users, and with us. [Come join us on Slack!](https://octopus.com/slack)

#### Contact

We have a dedicated support team to answer your questions and help you succeed. 

> If you need advice on licensing or upgrading your account, or have technical issues or complex CI/CD and integration questions, you can email our team at [customersuccess@octopus.com](mailto:customersuccess@octopus.com).

Reviewer: ```suggestion
If you need advice on anything from licensing or upgrading your account, to technical issues or complex CI/CD and integration questions, you can email our team at [customersuccess@octopus.com](mailto:customersuccess@octopus.com).
```

### Webinars

We run webinars that cover both fundamental and advanced topics. Octopus integrates with several cloud service providers and tools, and often our webinars hosted by Octopus and another CI/CD software tool provider. 

For newer users, this Octopus 101 webinar is a great place to start:

<iframe width="560" height="315" src="https://www.youtube.com/embed/mo0D4d5hFFU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

### Other useful links

#### Blog 

Our blog contains helpful information for new and existing users. Some topics we cover are:

- [General advice and insights about continuous integration](https://octopus.com/blog/difference-between-ci-and-cd)
- [Specific solutions to common customer problems](https://octopus.com/blog/chocolatey-powershell-and-runbooks)
- [An explanation of new features](https://octopus.com/blog/github-actions-for-octopus-deploy)
- [Tool specific guides](https://octopus.com/blog/deploying-ruby)
- [Kubernetes deployments](https://octopus.com/blog/10-pillars-kubernetes-deployments)

> 

Author: thanks fixed

You'll also find release notes, company announcements, and more. Customers tell us they enjoy engaging with the blog and find it helpful and informative, and we hope you do too. 

[Visit the blog page](https://octopus.com/blog/) to see the latest posts and search the topics that interest you.

#### Slack

Our Community Slack allows users to interact with other Octopus Deploy users, and with us. [Come join us!](https://octopus.com/slack)

#### Support

We have a dedicated support team to answer any questions you have. They are able to answer questions about Octopus Deploy and the software that it integrates with. Their goal is to give accurate and timely answers to all support questions. If you need help you can contact support at [support@octopus.com](mailto:support@octopus.com) for Octopus-specific technical issues, or you can email [advice@octopus.com](advice@octopus.com) for more complex CI/CD and integration questions.

#### Sales

> Every business is different and licensing needs may vary. Our sales team help businesses deploy the right Octopus Deploy services to meet their needs. If you need advice on licensing, pricing options, or upgrading your account, you can email our helpful sales team at [customersuccess@octopus.com](customersuccess@octopus.com).

Author: thanks fixed, I had a go at combining

#### Demos & samples

We hosted some demo and sample instances of Octopus Deploy:

- [This is a demo version of Octopus Deploy](https://demo.octopus.com/).
- [This is an instance with sample projects](https://samples.octopus.app/).

> Use these instances as sandbox environments to play with once you understand the basics. We encourage trial and experimentation. We'd love if you could let us know via [support](mailto:support@octopus.com) or [Slack](https://octopus.com/slack) if there's any samples you'd like to see.

Author: Hi @TeganAli, should this support@ be changed to customersuccess@ too?

Reviewer: @leecarlon can you please advise on this? It would be good to understand when to use each email address, as I'm a bit unclear on that at the moment. Thanks :)

Author: thanks fixed

## What next?

We hope this overview is helpful for getting started with Octopus Deploy. Octopus covers a large software surface area, and we want your experience as a new user to be as easy and useful as possible. 

We want you to test whether Octopus Deploy is right for you. If it’s not a good fit, rest assured that nobody is going to push otherwise; there’s no agenda. If it's the right solution for you, we want to give you all the resources you need to solve your CI/CD problems faster.  

> Contact our [support team](mailto:support@octopus.com) for any help you need. We look forward to helping you achieve your CI/CD goals.

Reviewer: Thanks Lee. @terence-octo can you please update to customersuccess@octopus.com as per Lee's advice? Thank you so much :)

> Contact our [support team](mailto:support@octopus.com) for any help you need. We look forward to helping you achieve your CI/CD goals.

Author: ```suggestion
Contact our [support team](mailto:customersuccess@octopus.com) for any help you need. We look forward to helping you achieve your CI/CD goals.
```

> Contact our [support team](mailto:support@octopus.com) for any help you need. We look forward to helping you achieve your CI/CD goals.

Author: Fixed :)

---
title: High level overview of Octopus Deploy
description: Find out about Octopus Deploy, resources to learn more and who to contact with questions.
author: terence.wong@octopus.com
visibility: public
published: 2021-10-20-1400
metaImage: blogimage-highleveloverviewofoctopusdeploy-2021.png
bannerImage: bblogimage-highleveloverviewofoctopusdeploy-2021.png
bannerImageAlt: overhead view of 6 people sitting around a betting table with octopus logo in the centre
isFeatured: false
tags:
 - Product
---

## Welcome

> Well done! You've deployed your first application with Octopus Deploy! 

Reviewer: ```suggestion
Well done! You deployed your first application with Octopus Deploy! 
```
